lib/membrane/compositor_element.ex

```diff
@@ -19,19 +19,21 @@ defmodule Membrane.VideoCompositor.CompositorElement do
   alias Membrane.RawVideo
   alias Membrane.VideoCompositor.Wgpu
 
-  def_options caps: [
-                spec: RawVideo.t(),
-                description: "Struct with video width, height, framerate and pixel format."
-              ],
-              real_time: [
-                spec: boolean(),
-                description: """
-                Set the compositor to real-time mode.
-                """,
-                default: false
-              ]
-
-  def_input_pad :input,
+  def_options(
+    caps: [
+      spec: RawVideo.t(),
+      description: "Struct with video width, height, framerate and pixel format."
+    ],
+    real_time: [
+      spec: boolean(),
+      description: """
+      Set the compositor to real-time mode.
+      """,
+      default: false
+    ]
+  )
+
+  def_input_pad(:input,
```

Comment:
Weird formating here too, please fix this

Reply:
fixed